lighthouse-treemap/app/src/main.js

```diff
@@ -309,11 +312,106 @@ class TreemapViewer {
     };
   }
 
+  createTable() {
+    const tableEl = TreemapUtil.find('.lh-table');
+    tableEl.innerHTML = '';
+
+    /** @type {Array<{name: string, resourceBytes: number, unusedBytes?: number}>} */
+    const data = [];
+    TreemapUtil.walk(this.currentTreemapRoot, (node, path) => {
+      if (node.children) return;
+
+      // Elide the first path component, which is common to all nodes.
+      let name;
+      if (path[0] === this.currentTreemapRoot.name) {
+        name = path.slice(1).join('/');
+      } else {
+        name = path.join('/');
```

Comment:
I did some stuff already to somewhat alleviate: the `//` is one, the other is trimming the selected bundle url.

perhaps if the name is a module in a bundle, the entire network URL part should be elided? a tooltip could then provide the url. the followup hover-on-row feature would also serve as a way to link a row to a particular bundle above.

Reply:
![image](https://user-images.githubusercontent.com/4071474/116147685-42756c00-a69d-11eb-8557-299cd2cbb5af.png)


`(bundle)` or `(sm)` or something else?

this change shortens things nicely. might still consider eliding long URLs of 3ps..

Reply:
on vc, connor also suggested reusing the color we have above.. so we could do a dot/square using that color.  i like that a lot. 

the current `(bundle)` prefix is fine, i don't love it, but whatever.

i really think the tooltip should include the module URL. it makes the tooltip HUGE but i hate when text gets elided because of narrow columns (like in DevTools network panel) and i don't have a quick way of hovering to see what the full thing is.